MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
```

Comment:
Please delete this line and the next one

Reply:
- [ ] I think it could be good to provide the reasons for your requirements, this will help the student understand why he needs to do what you suggest him to do 😉